server/server.js

```diff
@@ -4,35 +4,51 @@ const app = express();
 const port = process.env.PORT || 5000;
 
 const events = [
-
+  {id: 0,
```

Comment:
Are these changes already in master? You may need to merge master into this branch.

1. `git fetch` - Will fetch all the changes from the remote server
2. `git checkout client` - Will put you on the current branch
3. `git merge origin/master` - Will merge the remote master branch that you just fetched into the client branch